### DISCUSSION TITLE
Invalid Invite

### DISCUSSION THREAD

**Discussion by outlogic** [ramonvc/freegpt-webui#198] [General]
After clicking the https://discord.com/invite/hfmwJzctC6 link, I get the following error:
Invalid Invite
![index](https://github.com/ramonvc/freegpt-webui/assets/61180105/a9df306c-c66d-4410-a117-a83931c8568f)

And without the API key the only reply I get from the Chimera Version is the following:
<!doctype html> <html lang=en> <title>500 Internal Server Error</title> <h1>Internal Server Error</h1> <p>The server encountered an internal error and was unable to complete your request. Either the server is overloaded or there is an error in the application.</p>

![0](https://github.com/ramonvc/freegpt-webui/assets/61180105/3408feb0-196f-4fad-bbcf-34e8da4f5320)

**Comment by David-Roklem** [comment]
I encountered the same error. The project was working fine and then suddenly this error appeared. I did not make any changes to the code prior to this error occurance.
Have you been able to figure out the cause of the problem?

**Reply by outlogic** [reply]
Unfortunately I haven't found any solution to the problem. But the project wasn't working for me right from the start

**Reply by nqf** [reply]
I use this https://github.com/fantasy-peak/cpp-freegpt-webui
